Example/URBNSwiftyConvenience/ViewController.swift

```diff
@@ -7,12 +7,99 @@
 //
 
 import UIKit
+import URBNSwiftyConvenience
 
 class ViewController: UIViewController {
+    
+    var animateBorder: UIView?
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        let mondrian = UIView()
+        mondrian.backgroundColor = UIColor(white: 0.9, alpha: 1)
+        mondrian.translatesAutoresizingMaskIntoConstraints = false
+        
+        mondrian.urbn_topBorder = Border(color: .green, pixelWidth: 3, insets: UIEdgeInsets(top: 2, left: 12, bottom: 0, right: 0))
+        mondrian.urbn_bottomBorder = Border(color: .purple, pixelWidth: 5, insets: UIEdgeInsets(top: 0, left: 0, bottom: 5, right: 12))
+        mondrian.urbn_leadingBorder = Border(color: .red, pixelWidth: 7, insets: UIEdgeInsets(top: 10, left: 10, bottom: 0, right: 10))
+        mondrian.urbn_trailingBorder = Border(color: .orange)
```

Comment:
While swift supports name-spaced _types_, it doesn't support namespaces properties. If Apple adds these properties, it could cause issues so I kept it in.

but yeah, they’re ugly